dump.rdb

```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```

Comment:
This is strange. Not huge deal but if this could be removed that would be great.